rfc3164/parser.go

```diff
@@ -0,0 +1,45 @@
+package rfc3164
+
+import (
+	"sync"
+
+	syslog "github.com/influxdata/go-syslog/v2"
+)
+
+// parser represent a RFC3162 parser with mutex capabilities.
```

Comment:
Probably 3164 :)  I did take a minute and read a bit about [3162](https://tools.ietf.org/html/rfc3162) though

Reply:
Tnx